crates/runtime/src/traphandlers/macos.rs

```diff
@@ -434,7 +434,7 @@ unsafe extern "C" fn unwind(
         } else {
             None
         };
-        state.set_jit_trap(wasm_pc, wasm_fp, faulting_addr);
+        state.set_jit_trap(wasm_pc, wasm_fp, faulting_addr.into());
```

Comment:
Does this work on MacOS? Just on a Rust-compiling level, not related to the MTE error message functionality.

Reply:
Yes, works